src/_guides/language/language-tour.md

```diff
@@ -4008,14 +4008,14 @@ Future<String> lookUpVersion() async => '1.0.0';
 
 Note that the function's body doesn't need to use the Future API.
 Dart creates the Future object if necessary.
-
 If your function doesn't return a useful value,
 make its return type `Future<void>`.
 
-{% comment %}
-PENDING: add example here
+For an interactive introduction to using futures, `async`, and `await`,
+see the [asynchrony codelab](/codelabs/async-await).
```

Comment:
If we change the title of the codelab from "Asynchrony..." to "Asynchronous programming" (see above comment) we may also want to reconsider referring to the codelab as the "asynchrony" codelab. I'll wait to hear your thoughts before brainstorming further.